spec/Overview.html

```diff
@@ -7235,12 +7224,46 @@ <h5>Verify</h5>
                     |key| is "`P-256`", "`P-384`" or "`P-521`":
                   </dt>
                   <dd>
-                    <p>
-                      Perform the ECDSA verifying process, as specified in [[RFC6090]], Section 5.3, with |M| as the received
-                      message, |signature| as the received signature and using
-                      |params| as the EC domain parameters, and
-                      |Q| as the public key.
-                    </p>
+                    <ol>
+                      <li>
+                        <p>
+                          Let |n| be the smallest integer such that |n| * 8 is greater than
+                          the logarithm to base 2 of the order of the base point of the elliptic curve identified
+                          by |params|.
+                        </p>
+                      </li>
+                      <li>
+                        <p>
+                          If |signature| does not have a [= byte sequence/length =] of |n| * 2 bytes,
+                          then return false.
+                        </p>
+                      </li>
+                      <li>
+                        <p>
+                          Let |s1| be the result of performing the
+                          Octet-String-to-Integer Conversion,
+                          as specified in [[RFC6090]], Section 6.1,
+                          with the first |n| bytes of |signature| as the octet string |S|.
+                        </p>
+                      </li>
+                      <li>
+                        <p>
+                          Let |s2| be the result of performing the
+                          Octet-String-to-Integer Conversion,
+                          as specified in [[RFC6090]], Section 6.1,
+                          with the last |n| bytes of |signature| as the octet string |S|.
+                        </p>
+                      </li>
+                      <li>
+                        <p>
+                          Perform the ECDSA verifying process, as specified in [[RFC6090]],
+                          Section 5.4.3, with |m| as the message,
+                          (|s1|, |s2|) as the signature,
```

Comment:
Why call these `s1` and `s2` when the canonical name for them is `r` and `s`? `s1` and `s2` is very confusing in this context.

Reply:
...oh, I see. RFC 6090 uses `s1` and `s2` for "KT-I", and then observes that it is mathematically the same as ECDSA later. Sigh. Why did we (cryptography practioners) do this to ourselves?

In over a decade of working in this space, I've never seen any system refer to them as s1 and s2. It's always r and s.

Reply:
Myeah, fair enough. I've reverted to an earlier less invasive patch and pushed that, and will switch to citing another spec later. We already cite SEC1 in a few places, though probably citing FIPS 186-5 would be more consistent with the other algorithms..